Header header
DataPort[] states #array of Dataport messages for all dynamixels on bus